ramp-database/rampdb/model/submission.py

```diff
@@ -56,8 +56,82 @@
 
 
 class Submission(Model):
-    """An abstract (untrained) submission."""
-
+    """Submission table.
+
+    Parameters
+    ----------
+    name : str
+        The submission name.
+    event_name : str
+        The event name.
+    session : :class:`sqlalchemy.orm.Session`
+        The session to directly perform the operation on the database.
+
+    Attributes
+    ----------
+    id : int
+        The ID of the table row.
+    event_team_id : int
+        The event/team ID.
+    event_team : :class:`rampdb.model.EventTeam`
+        The event/team instance.
+    name : str
+        The name of the submission.
+    hash_ : string
+        A hash to identify the submission.
+    files : list of :class:`rampdb.model.SubmissionFile`
+        The list of the files associated with the submission.
+    submission_timestamp : datetime
+        The date and time when the submission was added to the database.
+    sent_to_training_timestamp : datetime
+        The date and time when the submission was sent for training.
+    training_timestamp : datetime
+        The date and time when the training finished.
+    contributivity : float
+        The contributivity of the submission.
+    historical_contributivity : float
+        The historical contributivity.
+    type : {'live' or 'test'}
+        The type of submission.
+    state : str
+        The state of the submission.
```

Comment:
do we have a list of possible states somewhere?

Reply:
On the top of the file in the enum. It is quite long to state is there I find. But the info is quite missing as well. Do you have a proposal?

Reply:
just stated that they are stored in that enum